azure-mgmt-eventhub/src/test/java/com/microsoft/azure/management/eventhub/EventHubTests.java

```diff
@@ -72,11 +72,11 @@ public void canManageEventHubNamespaceBasicSettings() {
 
         EventHubNamespace namespace2 = eventHubManager.namespaces()
                 .define(namespaceName2)
-                .withRegion(REGION)
-                .withExistingResourceGroup(RG_NAME)
-                // SDK should use Sku as 'Standard' and set capacity.capacity in it as 11
-                .withCurrentThroughputUnits(11)
-                .create();
+                    .withRegion(REGION)
+                    .withExistingResourceGroup(RG_NAME)
+                    // SDK should use Sku as 'Standard' and set capacity.capacity in it as 11
+                    .withCurrentThroughputUnits(11)
```

Comment:
Might be worth simplifying this to .withThroughputUnits() since "current" is redundant in this context. Does this method set a limit or a fixed capacity?

Reply:
There are two ThroughputUnit config here, the throughput currently set for the event hub and the max throughput service is allowed to expand to, in-order to distinguish between these two we made it explicit in the method names - `withCurrentThroughputUnits` and `withThroughputUnitsUpperLimit`

Reply:
sounds good.